models/discordactions.js

```diff
@@ -114,10 +114,41 @@ const updateDiscordImageForVerification = async (userDiscordId) => {
   }
 };
 
+const getNumberOfMemberForGroups = async (groups = []) => {
+  try {
+    if (!groups.length) {
+      return [];
+    }
+    const roleIds = groups.map((group) => group.roleid);
+
+    const snapshots = await memberRoleModel.where("roleid", "in", roleIds).get();
+    const roleCount = {};
+
+    snapshots.forEach((doc) => {
+      const roleToMemberMapping = doc.data();
+
+      if (roleCount[roleToMemberMapping.roleid]) {
+        roleCount[roleToMemberMapping.roleid] += 1;
+      } else {
+        roleCount[roleToMemberMapping.roleid] = 1;
+      }
```

Comment:
After considering both approaches, I believe the choice between using the ternary operator and the if-else statement here is a matter of personal preference. In my opinion, my approach may result in better readability, even though it requires more lines of code. While your approach is shorter and concise, it may slightly compromise readability.

For now, I will stick with my current implementation. However, if you strongly disagree and would like to discuss this further, please let me know, and we can have a discussion about it.

Thank you.